codeforboston/maple
# Migration GoalsAndMission to Bootstrap

# Summary
This PR covers the bootstrap migration of the Goals and Mission FAQ Page (just by virtue of the files we're touching, this also include the AboutPageCard migration)

Issues: #1464 and #1448 

# Checklist

- [na] On the frontend, I've made my strings translate-able.
- [na] If I've added shared components, I've added a storybook story.

# Screenshots

Current Page: 
<img width="1166" alt="Screenshot 2024-03-26 at 7 16 30 PM" src="https://github.com/codeforboston/maple/assets/2694761/11baef49-b1f5-4326-b5be-0fff8b580fec">
<img width="1125" alt="Screenshot 2024-03-26 at 7 16 36 PM" src="https://github.com/codeforboston/maple/assets/2694761/ea68f2ea-7c54-4292-8958-5a04c37409a9">
<img width="1112" alt="Screenshot 2024-03-26 at 7 16 42 PM" src="https://github.com/codeforboston/maple/assets/2694761/b632a542-aa1e-4052-b83a-5d03a495d03e">
<img width="417" alt="Screenshot 2024-03-26 at 7 16 52 PM" src="https://github.com/codeforboston/maple/assets/2694761/a3606a45-60fc-4dc7-ac75-0010aa541fec">
<img width="381" alt="Screenshot 2024-03-26 at 7 16 59 PM" src="https://github.com/codeforboston/maple/assets/2694761/6e1e8014-8734-4894-bf4f-5477eaa15914">


Updated Page:

<img width="1003" alt="Screenshot 2024-03-26 at 7 32 34 PM" src="https://github.com/codeforboston/maple/assets/2694761/fd042ee7-1e9d-4f92-be51-25062f5ccb55">
<img width="936" alt="Screenshot 2024-03-26 at 7 32 40 PM" src="https://github.com/codeforboston/maple/assets/2694761/d04cb859-7f28-4196-b9a9-ebebca3e76d4">
<img width="893" alt="Screenshot 2024-03-26 at 7 32 45 PM" src="https://github.com/codeforboston/maple/assets/2694761/a0ac2c33-c6a3-4871-aa8c-ec3d74428730">
<img width="425" alt="Screenshot 2024-03-26 at 7 32 54 PM" src="https://github.com/codeforboston/maple/assets/2694761/18a0bab8-8de9-49d3-933b-b18cb2928a5e">
<img width="382" alt="Screenshot 2024-03-26 at 7 32 59 PM" src="https://github.com/codeforboston/maple/assets/2694761/36296042-9420-4f9d-93f5-57d7a3be638a">
<img width="410" alt="Screenshot 2024-03-26 at 7 33 04 PM" src="https://github.com/codeforboston/maple/assets/2694761/2f4314bd-16b5-4e1f-93c1-1cda2670af0b">


# Known issues

N/A

# Steps to test/reproduce

1. Go to the mission and goals FAQ page: (https://www.mapletestimony.org/about/mission-and-goals on production)
1. Compare the dev site to production (at various screen sizes) and verify that they look right
